vuepress/docs/.vuepress/next.js

```diff
@@ -424,6 +429,16 @@ module.exports = {
                     path  + 'compose/creating-protected-resources.md'
                 ]
             },
+            {
+                title: 'Consume',
```

Comment:
Feels a little odd to have Consume Applications under Docs, but just plain Consume under tutorials. Maybe use the same language for both? Consume is the C that seems least intuitive to me so maybe Consume an Application is best, unless the parallel text with "Compose an Application" isn't good.